hatchery/src/world.rs

```diff
@@ -31,13 +33,17 @@ use crate::snapshot::{MemoryPath, Snapshot, SnapshotLike};
 use crate::storage_helpers::module_id_to_name;
 use crate::Error::PersistenceError;
 
+const DEFAULT_POINT_LIMIT: u64 = 1_000_000;
+const POINT_PASS_PERCENTAGE: u64 = 93;
+
 #[derive(Debug)]
 pub struct WorldInner {
     environments: BTreeMap<ModuleId, Env>,
     storage_path: PathBuf,
     events: Vec<Event>,
     call_stack: CallStack,
     height: u64,
+    limit: u64,
```

Comment:
The point limit does not belong to the world.

Reply:
Lets resolve this in favor of https://github.com/dusk-network/hatchery/issues/41

Reply:
postponed for later refactor.